meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```diff
@@ -33,7 +33,6 @@ RDEPENDS:${PN}-dbg = " \
 # it should be fixed in the upstream alsa recipe.
 
 RDEPENDS:${PN}-alsa = " \
-    packagegroup-base-alsa \
```

Comment:
We don't have alsa in `MACHINE_FEATURES` in zarhus-machine if I'm looking correctly. Is packagegroup-base-alsa added if it's only in `DISTRO_FEATURES`? Package exists only when it's in [MACHINE_FEATURES](https://github.com/openembedded/openembedded-core/blob/8b5c66c91d94f4c8521fe9443e65d86063dba5e5/meta/recipes-core/packagegroups/packagegroup-base.bb#L17)
but it's also in rdepends when it's in [`DISTRO_FEATURES`, `MACHINE_FEATURES` or `IMAGE_FEATURES`](https://github.com/openembedded/openembedded-core/blob/8b5c66c91d94f4c8521fe9443e65d86063dba5e5/meta/recipes-core/packagegroups/packagegroup-base.bb#L57)